doc/classes/XRInterface.xml

```diff
@@ -82,7 +107,19 @@
 		<member name="interface_is_primary" type="bool" setter="set_is_primary" getter="is_primary" default="false">
 			[code]true[/code] if this is the primary interface.
 		</member>
+		<member name="vr_play_area_mode" type="int" setter="set_play_area_mode" getter="get_play_area_mode" enum="XRInterface.PlayAreaMode" default="1">
```

Comment:
Should this be `xr_play_area_mode` instead of `vr_play_area_mode`?

Reply:
It's just the grouping, the property is actually "play_area_mode", but I do agree, I think the group should become "xr"

Reply:
Well the actual property in code is `vr_play_area_mode`. You don't need a `vr_` prefix if you want to group by "VR" in the inspector. The group prefix is optional and only used for cosmetic purposes, so you can have:
```
ADD_GROUP("XR", "")
ADD_PROPERTY(PropertyInfo(Variant::INT, "play_area_mode", PROPERTY_HINT_ENUM, "Unknown,3DOF,Sitting,Roomscale,Stage"), "set_play_area_mode", "get_play_area_mode");
```